modules/javafx.graphics/src/test/java/test/javafx/scene/layout/HeaderBarTest.java

```diff
@@ -275,30 +234,25 @@ void alignmentOfCenterChild_resizable_withNonEmptyLeadingAndTrailingChild(String
         "BOTTOM_CENTER, 450, 40, 100, 50",
         "BOTTOM_RIGHT, 740, 40, 100, 50"
     })
-    void alignmentOfCenterChild_notResizable_withNonEmptyLeadingAndTrailingChild(String arg) {
-        String[] args = arg.split(",");
+    void alignmentOfCenterChild_notResizable_withNonEmptyLeadingAndTrailingChild(
```

Comment:
minor suggestion:

long test names do not help much (and actually may cause problems, see https://bugs.openjdk.org/browse/JDK-8334497)

also rather inconvenient to see in the logs, or IDEs.  You can use comments for human-readability and short(er) method names.